### PR TITLE
Reindex narratives on any object update; fix a bug in an ES request

### PR DESCRIPTION
**Pull request by jayrbolton**
* [x] I incremented the version in the `VERSION` file
* [x] I updated `CHANGELOG.md`
* [ ] I added test coverage for any changes

### CODE REVIEW

File: .github/workflows/test-release.yml [hunk 1]
```diff
@@ -26,17 +26,17 @@ jobs:
     - name: Poetry installation
       run: poetry install
     - name: Run tests
-      run: scripts/run_tests
+      run: scripts/run_unit_tests
```
Comment (jayrbolton) [inline .github/workflows/test-release.yml:29]: Temporary change until all the other tests get wrangled again

Comment (jayrbolton) [reply .github/workflows/test-release.yml:29]: Going to do that in a follow-up PR

**Review by slebras — APPROVED** [review]
Looks good to me, Just need to make sure to get the default testing framework back up (in next PR like you mention)

File: .github/workflows/test-release.yml (hunk 1 above)
Comment (slebras) [reply .github/workflows/test-release.yml:29]: sounds good

**Pull request merged**